back/spec/acceptance/resident_native_survey_responses_spec.rb

```diff
@@ -195,6 +194,44 @@
         end
       end
 
+      describe 'with custom_field_values_params for geo fields' do
+        [
+          {
+            factory: :custom_field_point,
+            param_value: 'POINT (4.31 50.85)',
+            expected_stored_value: { 'type' => 'Point', 'coordinates' => [4.31, 50.85] }
+          },
+          {
+            factory: :custom_field_line,
+            param_value: 'LINESTRING (4.30 50.85, 4.660 51.15)',
+            expected_stored_value: { 'type' => 'LineString', 'coordinates' => [[4.30, 50.85], [4.660, 51.15]] }
+          },
+          {
+            factory: :custom_field_polygon,
+            param_value: 'POLYGON ((4.3 50.85, 4.31 50.85, 4.31 50.86, 4.3 50.85))',
+            expected_stored_value: {
+              'type' => 'Polygon', 'coordinates' => [[[4.3, 50.85], [4.31, 50.85], [4.31, 50.86], [4.3, 50.85]]]
+            }
+          }
+        ].each do |field_desc|
```

Comment:
(All good, but using rspec-parameterized is preferable to using loops for creating specs imo.)

Reply:
We could also extract `project` and `form` from the loop to speed things up.